2017/JULY/JULY17/EXPTREE/EXPTREE.py

```diff
@@ -0,0 +1,57 @@
+#__BY:sirjan13__
+
+
+def gcd(n,m):
+    if n == 0:
+        return m
+    return gcd(m % n, n)
+
+
+t1, t2=0, 0
+
+
+def egcd(a, b):
+    x, y, u, v = 0, 1, 1, 0 	 	 
+    while a != 0:
+        q, r = b//a, b % a
+        m, n = x-u*q, y-v*q
+        b, a, x, y, u, v = a, r, u, v, m, n
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpgf_vkzf9/2017/JULY/JULY17/EXPTREE/EXPTREE.py
+++ b/tmp/tmpgf_vkzf9/2017/JULY/JULY17/EXPTREE/EXPTREE.py
@@ -11,7 +11,7 @@
 
 
 def egcd(a, b):
-    x, y, u, v = 0, 1, 1, 0 	 	 
+    x, y, u, v = 0, 1, 1, 0
     while a != 0:
         q, r = b//a, b % a
         m, n = x-u*q, y-v*q
```